CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* We no longer show the "second tree" sidebar dropdown when there are no available options. The possible options are defined by [the charon/getAvailable API](https://docs.nextstrain.org/projects/auspice/en/stable/server/api.html) response and as such vary depending on the server in use. ([#1795](https://github.com/nextstrain/auspice/pull/1795))
```

Comment:
Thanks for noting the possible options can vary depending on the server! Prompted me to review how auspice does it internally with [findAvailableSecondTreeOptions](https://github.com/nextstrain/auspice/blob/4aa07ef829a6a03c3fc74d15f89a91a137b9951d/cli/server/getDatasetHelpers.js#L143C7-L143C37) and how nextstrain.org uses [convertManifestJsonToAvailableDatasetList](https://github.com/nextstrain/nextstrain.org/blob/05aba2bc93ef855de7b0c932b977d98c11b40408/src/endpoints/charon/parseManifest.js#L31).

Reply:
Yeah this whole area needs to be rethought at some stage -- the current hierarchical dropdown for changing datasets, the available second trees, how it relates to snapshots etc. There's [some design work around](https://docs.google.com/document/d/1CvE05kzGlys0sDYoT8RUKl-DYp7xJ__04skrKiaafbk/edit#heading=h.g9bzfnhkl7ss) but nothing concrete.